ui-core/src/participant/landing/sections/ParticipationDetailTemplate.tsx

```diff
@@ -3,15 +3,16 @@ import React from 'react'
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
 import { faClock } from '@fortawesome/free-regular-svg-icons'
 
-import { SectionConfig } from 'api/api'
-import { getSectionStyle } from 'util/styleUtils'
-import { withValidatedSectionConfig } from 'util/withValidatedSectionConfig'
-import { requireOptionalString } from 'util/validationUtils'
+import { SectionConfig } from '../../../types/landingPageConfig'
```

Comment:
Some of the import paths in this file and others can be shortened. I don't have strong feelings about this but since I've seen it come up in other PRs, is our style preference to always use the shortened path?